simiyu1/JavaDevsNairobi
# 155965831 display the github user list

#### What does this PR do?
 - Display fetched github users users
#### Description of Task to be completed?
 - implement adapterview in mainactivity
 - implement recyclerview in mainactivity
 - create transformation for rounded images

#### How should this be manually tested?
 - Clone this repository and run it on your IDE, ensure you have an internet connection. 
 - When the app launches, you should be able to see a list of github users

#### What are the relevant pivotal tracker stories?
 - [#155965831](https://www.pivotaltracker.com/story/show/155965831)

#### Screenshots
<img width="343" alt="screen shot 2018-10-09 at 07 00 45" src="https://user-images.githubusercontent.com/17185082/46646138-2407fd80-cb91-11e8-9424-74794affb276.png">

<img width="385" alt="screen shot 2018-10-09 at 07 00 21" src="https://user-images.githubusercontent.com/17185082/46646150-2f5b2900-cb91-11e8-9b92-ba48ea03f32e.png">
